src/hardware/voodoo.cpp

```diff
@@ -7291,15 +7268,15 @@ static bool Voodoo_GetRetrace() {
 static double Voodoo_GetVRetracePosition() {
 	// TODO proper implementation
 	const auto time_in_frame = PIC_FullIndex() - v->draw.frame_start;
-	const auto vfreq         = v->draw.vfreq;
-	if (vfreq <= 0) {
+	const auto frame_period_ms = v->draw.frame_period_ms;
+	if (frame_period_ms <= 0) {
```

Comment:
Nice, `frame_period_ms` tells me exactly what I'm dealing with, as opposed to the vague unknown quantity called `vfreq` 😄